databuilder/tests/unit/extractor/test_es_column_stats_extractor.py

```diff
@@ -87,11 +145,12 @@ def _get_extractor(self) -> Any:
 
         return extractor
 
-    def test_extractor_without_technical_data(self) -> None:
+    def _test_extractor_without_technical_data(self, es_version: str, indices: Dict, stats: Dict) -> None:
```

Comment:
maybe this should have different name now?

Reply:
Good point. What do you think of `_extract_without_technical_data`?